### ISSUE TITLE
Unblock 

### ISSUE THREAD

**Issue by Sudhindhra0202** [tachiyomiorg/tachiyomi-extensions#11375]
### Source information

All

### Steps to reproduce

I think IP address has been blocked so please can anyone unblock me and thank you

### Expected behavior

Can you please solve my problem

### Actual behavior

Not loading at all I have to use VPN to load

### Tachiyomi version

Latest update

### Android version

Android 12

### Other details

Can you solve my problem as soon as possible

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by loocool2** [comment]
If you can access it with a VPN, that's not a app issue.
